app/src/main/java/io/pslab/adapters/CustomAdapter.java

```diff
@@ -0,0 +1,96 @@
+package io.pslab.adapters;
+
+import android.content.Context;
+import android.text.Html;
+import android.text.method.LinkMovementMethod;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebView;
+import android.widget.BaseExpandableListAdapter;
+import android.widget.TextView;
+
+import java.util.HashMap;
+import java.util.List;
+
+import io.pslab.R;
+
+public class CustomAdapter extends BaseExpandableListAdapter {
```

Comment:
Use meaningful names for variables (Instead of "CustomAdapter" use something that makes sense). This applies to the rest of PR.

Reply:
## Changed CustomAdapter name
Refactored `CustomAdapter` to `FaqListAdapter`